### ISSUE TITLE
Web - Search Results - Incorrect Page Title

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#86]
The page TITLE should be "Search results for TERM GOES HERE - Comply Chain - United States Department of Labor" instead of only "Comply Chain". 

You should tell assistive technology users like screen reader users and search engines what the page is followed the application and then organization (DOL) and you could even add in ILAB spelled out before the DOL part.

The page TITLE format applies to all Comply Chain pages, so please fix accordingly.

Search Results - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/search

**Comment by Abychalil** [comment]
Title added as requested.. fixed

**Comment by jfc3-dol** [comment]
@Abychalil,

Thanks for correcting the screen TITLE. Will look at it when the next version/build is sent, and I  have time to test.

**Comment by jfc3-dol** [comment]
The page TITLE now reads "Search - Comply Chain - United States Department of Labor" instead of adding the extra text to tell the user what term they were attempting to find. An example is "Search results for TERM GOES HERE - Comply Chain - United States Department of Labor".

I'm going to close the issue because at least it tells the user they are on the search page now.
